singularity/Singularity.amd_aocl

```diff
@@ -239,16 +239,16 @@ From: fedora:41
     cd /opt/hdf5
     rm -r hdf5-1.*
     export CC=`which gcc`
-	export CXX=`which g++`
+    export CXX=`which g++`
 
     if [ $HAS_CUDA = true ]; then
         #
-        # Install CUDA 11.5
+        # Install CUDA 12.6
         #
-        sudo dnf -y config-manager addrepo --from-repofile=http://developer.download.nvidia.com/compute/cuda/repos/fedora41/x86_64/cuda-fedora41.repo
+        sudo dnf -y config-manager addrepo --from-repofile=https://developer.download.nvidia.com/compute/cuda/repos/fedora39/x86_64/cuda-fedora39.repo
```

Comment:
I'm experimenting with which Fedora version to use, to same here; let's not change versions.